app/models/king.rb

```diff
@@ -1,2 +1,7 @@
 class King < Piece
+
+  def valid_move?(x,y)
+    y > y_coord + 1 || y < y_coord - 1 || x > x_coord + 1 || x < x_coord - 1 ? false : true
```

Comment:
What do `y > y_coord + 1` and `y < y_coord - 1` and same for `x` have in common? Thinking about finding a way to combine them in one statement.. Any ideas 😄

Reply:
Another thing: what if we put in `(-10,99)`? maybe we should also handle the case of being off board?

Reply:
@ilyakrasnov I'm just not seeing it. I've fixed for cases of moving off the board but it just made for more || statements. I just can't figure out how to combine them when the cases are specific to each variable used.